lib/kennel/models/record.rb

```diff
@@ -179,8 +225,14 @@ def resolve_link(sought_tracking_id, sought_type, id_map, force:)
       end
 
       # let users know which project/resource failed when something happens during diffing where the backtrace is hidden
-      def invalid!(message)
-        raise ValidationError, "#{tracking_id} #{message}"
+      def invalid!(tag, message = nil)
+        if message.nil?
+          message = tag
+          tag = :default_error_tag
+          Kennel.out.puts "I: untagged validation error '#{message}'" if ENV["SHOW_DISABLED_VALIDATION"]
```

Comment:
maybe just leave it in the system as untagged and we can use some hacky code to find all the untagged places later

Reply:
I'm opting now for not having the "default tag" concept. On upgrade, overriders (e.g. zendesk-kennel) will have to decide whether each error is either skippable (in which case give it a tag), or not (in which case add the tag as `nil`).